annotations/docker-annotations/src/main/java/io/ap4k/docker/decorator/ApplyRegistryToImageDecorator.java

```diff
@@ -21,24 +21,28 @@
 import io.ap4k.kubernetes.decorator.ApplyImageDecorator;
 import io.ap4k.kubernetes.decorator.Decorator;
 import io.ap4k.utils.Images;
+import io.ap4k.Resources;
 
 public class ApplyRegistryToImageDecorator extends Decorator<DeploymentBuilder> {
 
-  private final String name;
-  private final String image;
+  private final String registry;
+  private final Resources resources;
 
-  public ApplyRegistryToImageDecorator(String registry, String group, String name, String version) {
-    this.name = name;
-    this.image = Images.getImage(registry, group, name, version);
+  public ApplyRegistryToImageDecorator(Resources resources, String registry) {
+    this.registry=registry;
+    this.resources=resources;
+    String a = "this is shaaait";
```

Comment:
I am guessing this was left over by mistake?

Reply:
LoL